PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -978,6 +978,7 @@ private void ThisAddInSlideSelectionChanged(PowerPoint.SlideRange sldRange)
                     prev = presentation.Slides[slideIndex - 1];
                 }
             }
+            Ribbon.RefreshRibbonControl("CreateMenu");

```

Comment:
Could this be called "CreateTooltipMenuButton" instead, if CreateMenu is only relevant to TooltipsLab? This button name seems to be inconsistent with the rest.

Reply:
This naming actually follows the convention for the rest of the labs. "CreateMenu" is actually a Menu, not a Button. "CreateMenu" expands into 3 buttons, "CreateTooltipButton", "CreateCalloutButton" and "CreateTriggerButton".